portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase

```diff
@@ -436,9 +847,81 @@ definition {
 			successPageTitle = "Thank you.");
 	}
 
-	@description = "Verify that a Text Field can require Confirmation"
+	@description = "LPS-125860 - Verify that Confirmation field property is working as expected"
 	@priority = "5"
 	test CanRequireConfirmation {
+		property portal.acceptance = "true";
```

Comment:
@leticiachiu, should it be in the acceptance suite? Can you please confirm with Diogo?

Reply:
I just moved Forms#SubmitFormWithRequiredConfirmationTextField to FormsTextField.testcase, so I kept the property.. is that right, @diogocordeiro?

Reply:
Makes sense, @leticiachiu! :slightly_smiling_face: